demo/api/schema.gql

```diff
@@ -909,6 +926,27 @@ enum ProductTagSortField {
   updatedAt
 }
 
+input ProductVariantFilter {
+  name: StringFilter
+  product: ManyToOneFilter
```

Comment:
This shouldn't exist as product is a root arg, right?

Reply:
true, that has no use. Removed.